Tasks/JavaToolInstaller/task.json

```diff
@@ -62,8 +62,8 @@
             "type": "filePath",
             "label": "JDK file",
             "required": true,
-            "visibleRule": "jdkSourceOption == LocalDirectory",
-            "helpMarkDown": "Path to where the compressed JDK is located. The path could be in your source repository or a local path on the agent."
+            "visibleRule": "jdkSourceOption == FilePath",
```

Comment:
This task is in preview, but I realized that this will actually break existing definitions.  I will handle the old value of the setting in the code.  We can get rid of it with the next major version update of this task.